pyomo/contrib/mindtpy/tests/test_mindtpy_lp_nlp.py

```diff
@@ -121,19 +121,33 @@ def test_lazy_OA_Proposal(self):
                           TerminationCondition.optimal)
             self.assertAlmostEqual(value(model.obj.expr), 0.66555, places=2)
 
+    def test_lazy_OA_ConstraintQualificationExample(self):
+        with SolverFactory('mindtpy') as opt:
+            model = ConstraintQualificationExample()
+            print('\n Solving ConstraintQualificationExample with LP/NLP')
+            results = opt.solve(model, strategy='OA',
+                                mip_solver=required_solvers[1],
+                                nlp_solver=required_solvers[0],
+                                single_tree=True
+                                )
+            # TODO: constraint qualification hold true in this case
```

Comment:
I guess we can remove this TODO. We can also add an assertion for a feasible solution instead of optimal (uncomment and correct next line).